admin/test/test_packaging.py

```diff
@@ -535,6 +534,24 @@ def test_version_found(self):
         step.run()
         self.assertEqual(test_package.version, step.version)
 
+    def test_version_found(self):
+        """
+        ``GetPackageVersion`` assigns the exact version of a found package to
+        its ``version`` attribute.
```

Comment:
:question: Change this docstring to reference the `pip` version string manipulation which this test shows that GetPackageVersion avoids.